pkg/routeagent_driver/handlers/ovn/handler.go

```diff
@@ -144,6 +144,12 @@ func (ovn *Handler) RemoteEndpointCreated(endpoint *submV1.Endpoint) error {
 	}
 
 	if ovn.isGateway {
+		for _, subnet := range endpoint.Spec.Subnets {
+			if err = ovn.addNoMasqueradeIPTables(subnet); err != nil {
+				return errors.Wrap(err, "error adding no-masquerade rules")
```

Comment:
Please include the subnet (or the rule) for which the error occurred.

Reply:
Done.